DOCS/interface-changes/bluray-protocol.txt

```diff
@@ -0,0 +1 @@
+`bluary://` will now use the ffmpeg protocol, use `bd://` to continue using the mpv protocol
```

Comment:
What are the implications of this? What is the difference?

Reply:
A BD disc would be passed directly to ffmpeg's protocol (assuming your ffmpeg was built with it of course) instead of using mpv's custom one. I didn't actually test it though.

Reply:
Ok, since `bluray://` was not primary option in docs, should be fine. Maybe mention this in the docs that `bluray://` starting with 0.39 will behave differently. Instead of just removing alias mention.

Reply:
Added a note.